<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="GraphicsView" name="graphicsView">
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open_insole_file"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_set_cutting_flat"/>
    <addaction name="action_cutting_mesh"/>
    <addaction name="action_range_projector"/>
   </widget>
   <widget class="QMenu" name="menu_edit_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about_Qt"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_edit_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_insole_file"/>
   <addaction name="action_set_cutting_flat"/>
   <addaction name="action_cutting_mesh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open_insole_file">
   <property name="icon">
    <iconset resource="../resources/sources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Input insole model</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;打开鞋垫模型，导入程序……&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_set_cutting_flat">
   <property name="icon">
    <iconset resource="../resources/sources.qrc">
     <normaloff>:/icons/flat.png</normaloff>:/icons/flat.png</iconset>
   </property>
   <property name="text">
    <string>设置切割平面</string>
   </property>
   <property name="toolTip">
    <string>设置切割平面 f = a * x + b * y + c * z + d</string>
   </property>
  </action>
  <action name="action_about_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_cutting_mesh">
   <property name="icon">
    <iconset resource="../resources/sources.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>平面与模型求交</string>
   </property>
  </action>
  <action name="action_range_projector">
   <property name="text">
    <string>range projector</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/sources.qrc"/>
 </resources>
 <connections/>
</ui>
